xbmc/interfaces/json-rpc/SettingsOperations.cpp

```diff
@@ -684,6 +695,26 @@ bool CSettingsOperations::SerializeSettingAddon(const CSettingAddon* setting, CV
   return true;
 }
 
+bool CSettingsOperations::SerializeSettingDate(const CSettingDate* setting, CVariant &obj)
```

Comment:
just because it piqued my curiosity; why isn't this done as CSettingDate::Serialize ?